java/core-scala/src/main/scala/io/pdal/pipeline/ReaderTypes.scala

```diff
@@ -0,0 +1,73 @@
+/******************************************************************************
+  * Copyright (c) 2017, hobu Inc.  (info@hobu.co)
```

Comment:
I think the copyright on these Scala bits should be to Azavea. Happy to sign a CLA to allow for things like future relicensing.